db/migrate/20230607055537_create_likes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :user_id
```

Comment:
- [ ] @adamilare great work adding the likes table, however, the reference to the user should be authorid as it is in the ERD diagram,
![image](https://github.com/adamilare/railsblog/assets/61408860/c711dff7-6c88-4f27-91bd-e2b195daa408)

Reply:
@danmainah , thank you for your review.

That image is different from the one provided in the activity. Can you please provide me the link to your reference?
As per linked ERD, only Post has 'AutherId'; Like and Comment both have 'UserId'

[Given ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png)